src/load_inputs/load_generators_data.jl

```diff
@@ -501,4 +518,27 @@ function load_vre_stor_data!(inputs_gen::Dict, setup::Dict, path::AbstractString
 		inputs_gen["dfVRE_STOR"] = DataFrame()
 	end
 	summarize_errors(error_strings)
-end
\ No newline at end of file
+end
+
+function get_resources_which_can_be_retired(df::DataFrame)::Set{Int64}
+    if string(:Can_Retire) in names(df)
+        retirable = df[df.Can_Retire.==1, :R_ID]
+
+        invalid_newbuild = df.New_Build.==-1
+        if any(invalid_newbuild)
+            @info "Deprecated '-1' entries in the 'New_Build' column.
```

Comment:
Suggest throwing an error (or at least strong warning) instead of an @info if users do have the Can_Retire column.